pages/common/git-am.md

```diff
@@ -0,0 +1,16 @@
+# git am
+
+> Apply patch files. Useful when receiving commits via email.
+> See also `git format-patch`, which can generate patch files.
+
+- Apply a patch file:
+
+`git am {{path/to/file.patch}}`
+
+- Abort the process of applying a patch file:
+
+`git am --abort`
+
+- Apply as much fo a patch file as possible, saving failed hunks to reject files:
```

Comment:
typo at "fo" ? "hunks" ?

Reply:
Ah, good spot!

And that's the technical name it uses in the man page * shrugs *